docker-compose-full.yml

```diff
@@ -100,6 +107,57 @@ services:
     volumes:
       - ../generic-tooling-source:/opt/generic-tooling-target
 
+  language-server-proxy:
+    image: exercism/language-server-proxy
+    build:
+      context: ../language-servers
+      dockerfile: proxy.Dockerfile
+    ports:
+      - 3023:3023
+    environment:
+      RUBY_LANGUAGE_SERVER_HOST: ruby-language-server
+      RUBY_LANGUAGE_SERVER_PORT: 7658
+      JAVASCRIPT_LANGUAGE_SERVER_HOST: javascript-language-server
+      JAVASCRIPT_LANGUAGE_SERVER_PORT: 2538
+      CSHARP_LANGUAGE_SERVER_HOST: csharp-language-server
+      CSHARP_LANGUAGE_SERVER_PORT: 8291
+    volumes:
+      - ../language-servers:/opt/exercism/language-server-proxy/current
+
+  ruby-language-server:
+    image: exercism/ruby-language-server
+    build:
+      context: ../language-servers
+      dockerfile: ruby.Dockerfile
+    environment:
+      PROXY_HOST: language-server-proxy
+      PROXY_PORT: 4444
+    volumes:
+      - ../language-servers:/opt/exercism/ruby-language-server/current
+
+  javascript-language-server:
+    image: exercism/javascript-language-server
+    build:
+      context: ../language-servers
+      dockerfile: javascript.Dockerfile
+    environment:
+      PROXY_HOST: language-server-proxy
+      PROXY_PORT: 4444
+    volumes:
+      - ../language-servers:/opt/exercism/javascript-language-server/current
+
+  csharp-language-server:
+    image: exercism/csharp-language-server
+    tty: true
+    build:
+      context: "../language-servers"
+      dockerfile: csharp.Dockerfile
+    environment:
+      PROXY_HOST: language-server-proxy
+      PROXY_PORT: 4444
+    volumes:
+      - ../language-servers:/opt/exercism/csharp-language-server/current
```

Comment:
I initially attempted to use a generic template like I did with the tooling, but the main issues with that was that the `csharp-language-server` has an additional `tty` property. We probably don't want to set this property for the other language servers. As the existing code does not support merging two definitions, I've postponed this for a bit.

Reply:
@ErikSchierboom We can actually do without the `tty` property. I've been only using that to test. Sorry, I wasn't able to catch that.